app/login/login.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component} from "@angular/core";
+import { FormBuilder, Validators, FormGroup } from "@angular/forms";
+import { RouterModule, Router } from '@angular/router';
+
+import { LoginService } from "./login.service";
+
+@Component({
+  selector: "login",
+  templateUrl: "app/login/login.component.html",
+  styleUrls:  ["app/login/login.component.css"]
+})
+export class LoginComponent {
+	loginForm :FormGroup;
+	user: {};
+
+    constructor(
+    	public fb: FormBuilder,
+    	private router: Router,
+    	private loginService: LoginService
+	) {
+    	this.loginForm = this.fb.group({
+		   	username: ["", Validators.required],
+	    	password: ["", Validators.required];
+	    });
+	}
+ 
+	doLogin(value) {
+		this.loginService.getUsers()
+		.subscribe(
+            data => {
+            	if(this.chkUser(value, data)) {
+            		this.router.navigate(['/trainings']);
+            	}
+            	else {
+            		alert('Enter Valid Credentials');
+            	}
+            },
+            error => {
+            	//Handle Error here
+            }
+        );
+	}
+
+	chkUser(userValue, users) {
+		this.user = users.filter(element => {
```

Comment:
=> defines a function type where the arguments are to the left of the => and the return type is on the right